From f8af46cdd209658c44385daa47689b1e87661443 Mon Sep 17 00:00:00 2001
From: Harsh J <harsh@cloudera.com>
Date: Tue, 17 Mar 2015 11:06:35 +0530
Subject: [PATCH 1298/2848] MAPREDUCE-6100. replace
 mapreduce.job.credentials.binary with
 MRJobConfig.MAPREDUCE_JOB_CREDENTIALS_BINARY for
 better readability. Contributed by Zhihai Xu.

(cherry picked from commit f222bde273cc10a38945dc31e85206a0c4f06a12)

Change-Id: I431fa73335083411ebd9fb528326ad67c0e8af0e
---
 .../org/apache/hadoop/mapreduce/JobSubmitter.java  |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/JobSubmitter.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/JobSubmitter.java
index 12f62bf..276f755 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/JobSubmitter.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/JobSubmitter.java
@@ -382,7 +382,7 @@ private void readTokensFromFiles(Configuration conf, Credentials credentials)
   throws IOException {
     // add tokens and secrets coming from a token storage file
     String binaryTokenFilename =
-      conf.get("mapreduce.job.credentials.binary");
+      conf.get(MRJobConfig.MAPREDUCE_JOB_CREDENTIALS_BINARY);
     if (binaryTokenFilename != null) {
       Credentials binary = Credentials.readTokenStorageFile(
           FileSystem.getLocal(conf).makeQualified(
-- 
1.7.9.5

